src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+/* eslint-disable no-nested-ternary */
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age !== undefined && (
+      <p className="Person__age">I am {person.age}</p>
+    )}
+    <p className="Person__sex">{person.sex}</p>
+    <p className="Person__isMarried">
+      {person.isMarried
+        ? person.sex === 'm'
+          ? `${person.partnerName} is my wife`
+          : `${person.partnerName} is my husband`
+        : 'I am not married'}
+    </p>
+    <p className="Person__partner">
+      {person.isMarried && person.partnerName
+        ? person.sex === 'm'
+          ? `${person.partnerName} is my wife`
+          : `${person.partnerName} is my husband`
+        : 'not married'}
```

Comment:
The logic for displaying marital status and partner information is duplicated. You can remove one of these blocks to avoid redundancy. Consider keeping one block that handles both the marital status and partner information correctly.